config/default_proffieboard_config.h

```diff
@@ -12,14 +12,15 @@
 #include "proffieboard_config.h"
 #define NUM_BLADES 2
 #define NUM_BUTTONS 2
-#define VOLUME 1000
+#define VOLUME 100
```

Comment:
why is this change here?

Reply:
For debugging late at night  :-)
(should be reset to correct version once you are satisfied with all the adjustments for dim_blade pull request)